Riot/Modules/Analytics/AnalyticsUIElement.swift

```diff
@@ -18,15 +18,24 @@ import AnalyticsEvents
 
 /// A tappable UI element that can be tracked in Analytics.
 @objc enum AnalyticsUIElement: Int {
-    case removeMe
+    case roomThreadListButton
+    case roomThreadSummaryItem
+    case threadListThreadItem
+    case threadListFilterItem
     
     /// The element name reported to the AnalyticsEvent.
     var name: AnalyticsEvent.Interaction.Name {
         switch self {
         // Note: This is a test element that doesn't need to be captured.
         // It can be removed when some elements are added that relate to mobile.
```

Comment:
Thanks for removing `removeMe`. This comment can go too :)

Reply:
Yep 👍